packages/core/src/components/link/link.css

```diff
@@ -0,0 +1,60 @@
+.hds-link {
+  --link-visited-color: #551a8b;
+  --link-color: var(--color-bus);
+  text-decoration: underline;
+  color: var(--link-color);
+  box-sizing: border-box;
+  border: 3px solid transparent;
+}
+
+.hds-link:visited {
+  color: var(--link-visited-color);
+}
+.hds-link:visited svg g path {
```

Comment:
This was needed for Safari. Without this the external link icon did not get the visited link color when the link had already been visited on the react version of the component. It might be a good idea to add a comment here about it? Also not sure if this is the best way to solve the issue.

Reply:
There might be something wrong with the HDS icon implementation 🤔
But some comments would be nice 👍

Reply:
Or this might be a Safari bug as well 🤷

Reply:
- [ ] add comment